### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by mikaelhg**
Build the project in Docker, produce a runnable image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM oracle/graalvm-ce:19.3.1-java8 AS BUILD
+RUN curl -o - -s http://www.nic.funet.fi/pub/mirrors/apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz | tar xz -C /usr/local
+RUN gu install native-image
+COPY . /build
+WORKDIR /build
+RUN /usr/local/apache-maven-3.6.3/bin/mvn
+
+FROM ubuntu:20.04
+RUN apt-get -qq update && apt-get -y -qq install libxi6 libxtst6 libxrender1
```
Comment (vektory79) [inline Dockerfile]: 1. These packages are not enough.
2. The apt-get generate garbage during its work. Therefore that garbage should be cleared.

I think, this line should be rewritten like this:

```suggestion
RUN apt-get -qq update \
 && apt-get -y -qq install libxi6 libxtst6 libfreetype6 libpng16-16 libfontconfig1 libjpeg8 fonts-dejavu-extra graphviz \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment (mikaelhg) [reply Dockerfile]: BTW, thanks for introducing the Suggested change functionality to me, I hadn't seen it before.

**Comment by vektory79** [comment]
Thanks for idea with docker!

**Comment by mikaelhg** [comment]
Do you think it would be possible to use Github Actions to build this for Windows? I have a bunch of Windows developer colleagues.

**Comment by vektory79** [comment]
Sadly, I'm a Linux user. And have no any imagine about support and building of the native-image in Windows.

If anyone make the PR with this support, I will be glad.

**Pull request merged**